packages/gasket-utils/lib/warn-if-outdated.js

```diff
@@ -1,4 +1,4 @@
-const { runShellCommand } = require('@gasket/utils');
+const { runShellCommand }  = require('./index');
```

Comment:
`npm run lint:fix` maybe?

```suggestion
const { runShellCommand } = require('./index');
```